ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -566,7 +593,8 @@
       },
       {
         "UserRoles": [
-          "caseworker-civil-admin"
+          "caseworker-civil-admin",
+          "caseworker-civil-judge"
```

Comment:
Not sure if this would be required - please confirm if the business wants this!

Reply:
I have changed the access to read only for now whilst pending business confirmation